src/scss/_utilities.scss

```diff
@@ -220,4 +220,22 @@
   .u-width--full {
     width: 100% !important;
   }
+
+  $nav-theme-colors: (
+    "default": $colors--dark-theme--background-default,
+    "bark": #585841,
+    "sage": #4e5f51,
+    "olive": #3d5f11,
+    "viridian": #025a3d,
+    "prussian_green": #225d5c,
+    "blue": #0060bf,
+    "purple": #7764d8,
+    "magenta": #974097,
+    "red": #a71b33,
+  );
+  @each $name, $color in $nav-theme-colors {
+    .is-maas-#{$name} {
+      background-color: $color !important;
+    }
+  }
```

Comment:
This is really great! Really clean implementation and allows us to theme other stuff in the future as well. Nice one!